packages/indexer-agent/src/network.ts

```diff
@@ -223,7 +223,8 @@ export class Network {
       if (
         error.message.includes(
           'Transaction with the same hash was already imported',
```

Comment:
Now that I see this, the error "Transaction with the same hash was already imported" means that the transaction we sent is already on the mempool, so maybe in that case we should skip and just wait for it? if we change the nonce we risk to duplicate the transaction